lib/internal/test_runner/test.js

```diff
@@ -560,7 +561,7 @@ class Test extends AsyncResource {
         const map = lazyFindSourceMap(this.loc.file);
         const entry = map?.findEntry(this.loc.line - 1, this.loc.column - 1);
 
-        if (entry !== undefined) {
+        if (entry !== undefined && ObjectKeys(entry).length > 0) {
```

Comment:
I think it would be useful for you to open a separate PR that contains only the following things:

- The changes in this file. I don't think this is the exact code we'll want to merge, but it is its own issue.
- A new unit test that currently fails on `main` but passes with these changes.

Reply:
Done!

https://github.com/nodejs/node/pull/55231.

I tried to come up with a different change than `ObjectKeys(entry).length > 0` but couldn't figure out a better approach since `findEntry` might return an empty object https://github.com/nodejs/node/blob/main/lib/internal/source_map/source_map.js#L213-L218.


On this PR, if you guys don't mind, I'd like to keep code related to coverage collection fix when `--enable-source-maps` is passed (https://github.com/nodejs/node/pull/55228/files#diff-0e00da74475521ee48fdbfb1d8e6a6a7458bc8c70957e39b6fa1b0213509fa4dR426-R432).

Thanks for the suggestion!